docs/GCE.md

```diff
@@ -1,63 +1,110 @@
 # Running calico-docker on GCE
-Calico runs on the Google Compute Engine (GCE), but there are a few tweaks required to the main Getting Started instructions.  The following instructions show the full power of the Calico routing and security model on GCE (and allow GCE to be used for testing)
+Calico runs on the Google Compute Engine (GCE), but there are a few tweaks required to the main Getting Started instructions.  The following instructions show the full power of the Calico routing and security model on GCE (and allow GCE to be used for testing).
 
 ## Getting started
-These instructions assume a total of three GCE hosts running CoreOS. One to run an etcd "cluster", and then two compute nodes.
-Documentation on running CoreOS on GCE is [here](https://coreos.com/docs/running-coreos/cloud-providers/google-compute-engine/)
+These instructions assume a total of three GCE hosts running CoreOS. Documentation on running CoreOS on GCE is [here](https://coreos.com/docs/running-coreos/cloud-providers/google-compute-engine/)
 
 Download and install GCE and login to your account. Full documentation on how to install gcloud are given [here](https://cloud.google.com/compute/docs/gcloud-compute/).
 ```
 curl https://sdk.cloud.google.com | bash
 gcloud auth login
 ```
-
 Also, create a project through the GCE console and set that as the default for gcloud.
 ```
 gcloud config set project PROJECT
 ```
-
 And set a default zone
 ```
 gcloud config set compute/zone us-central1-a
 ```
+## Setting up GCE networking
+GCE blocks traffic between hosts by default, which prevents Calico traffic from flowing.  Run the following command to allow traffic to flow between containers:
```

Comment:
This line is a bit confusing to me - It says that GCE blocks traffic between hosts, but then says that this rule allows traffic to flow between containers (which may not be on different hosts).

Maybe just say "allow traffic to flow between containers on different hosts" instead?